base/shell/cmd/lang/zh-CN.rc

```diff
@@ -148,7 +149,7 @@ DELAY [/m]n\n\n\
                G  分组目录在前                    -  反转顺序的前缀\n\
   /P          在每屏幕信息输出后停顿。\n\
   /Q          显示文件的所有者。\n\
-  /R          Displays alternate data streams of the files.\n\
+  /R          显示文件的 NTFS 数据流。\n\
```

Comment:
They indeed refer to these, but the concept of alternate data streams is not specific just for NTFS. So having some translation that does not refer to "NTFS" would be better.

Reply:
ok, I will change a way to translate it🤣